Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -1725,14 +1726,6 @@ bool updateBondsModifiedByReaction(
         molModified = true;
       }
     }
-
-    if (rAtom->getAtomicNum() != pAtom->getAtomicNum()) {
-      atom->setAtomicNum(pAtom->getAtomicNum());
-      molModified = true;
-    }
-    if (ReactionRunnerUtils::updatePropsFromImplicitProps(pAtom, atom)) {
-      molModified = true;
-    }
```

Comment:
This block of code should never have been here. It's just duplicating the work which is being done at l1584